templates/taxi/car_detail.html

```diff
@@ -19,4 +19,15 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  {% if request.user in car.drivers.all %}
```

Comment:
you use here backward relationship but did you fetched it from db? because otherwise here will be N+1 issue